components/blocks/Pricing.tsx

```diff
@@ -211,42 +214,56 @@ export function PricingBlock({ data }) {
+
+        <div className="pt-2 max-w-7xl mx-auto flex justify-center">
           <FreeTier data={data} />
         </div>
-        <div className="pt-10 px-4 pb-6 text-center">
-          <TinaMarkdown content={data.intro} components={pricingComponents} />
-        </div>
-        <div className="flex justify-center pb-6">
-          <div className="bg-gradient-to-br font-tuner from-white/25 via-white/50 to-white/75 shadow-2xl rounded-full gap-16 relative w-max">
-            <div
-              className={`absolute top-0 left-0 w-1/2 h-full bg-gradient-to-br from-blue-300 via-blue-400 to-blue-500 rounded-full transition-transform duration-500 ease-in-out ${
-                isMonthly
-                  ? 'transform translate-x-0'
-                  : 'transform translate-x-full'
-              }`}
-            ></div>
-            <div className="relative flex z-10">
-              <button
-                className={`px-10 py-2 w-1/2 z-20 transition-colors duration-500 ${
-                  isMonthly ? 'text-white' : 'text-black'
-                }`}
-                onClick={() => setIsMonthly(true)}
-              >
-                Monthly
-              </button>
-              <button
-                className={`px-10 py-2 w-1/2 z-20 transition-colors duration-500 ${
-                  !isMonthly ? 'text-white' : 'text-black'
+        <div className="flex justify-center md:justify-start pt-10">
+          <div className="flex flex-col sm:space-y-4 md:flex-row md:items-center">
+            <div className="bg-gradient-to-br font-tuner from-white/25 via-white/50 to-white/75 shadow-md rounded-full gap-16 relative w-max">
+              <div
+                className={`absolute top-0 left-0 w-1/2 h-full bg-gradient-to-br from-blue-300 via-blue-500 to-blue-700 rounded-full transition-transform duration-500 ease-in-out border-4 border-white  ${
+                  isMonthly
+                    ? 'transform translate-x-0'
+                    : 'transform translate-x-full'
                 }`}
-                onClick={() => setIsMonthly(false)}
-              >
-                Annually
-              </button>
+              ></div>
+              <div className="relative leading-none flex z-10">
+                <button
+                  className={`px-10 py-4 w-1/2 z-20 transition-colors  duration-500 ${
+                    isMonthly ? 'text-white' : 'text-blue-500'
+                  }`}
+                  onClick={() => setIsMonthly(true)}
+                >
+                  Monthly
+                </button>
+                <button
+                  className={`px-10 py-2 w-1/2 z-20 transition-colors  duration-500 ${
+                    !isMonthly ? 'text-white' : 'text-blue-500'
+                  }`}
+                  onClick={() => setIsMonthly(false)}
+                >
+                  Annually
+                </button>
+              </div>
+            </div>
```

Comment:
This code is pretty noisy.
Consider moving it to a component
e.g. `PillSwitch`

Reply:
done

Reply:
This is not clear.

I think you meant

"Not done - I'm happy with how it is"